cypress/integration/a11y.spec.js

```diff
@@ -147,7 +147,7 @@ describe('Check accessiblity', () => {
 
         context('eApp applications section', () => {
             function selectFirstApplication() {
-                get('#previousApplications .appRef .govuk-link')
+                get('#dashboard-results tbody .govuk-link')
```

Comment:
Fair point. This is quite a tough case because the item I want to select isn't static, it's sort of dynamic

<img width="798" alt="Screenshot 2022-02-11 at 17 50 54" src="https://user-images.githubusercontent.com/1377253/153643231-a8b3c5f9-402d-4b2c-9882-3902abc78d7a.png">

So I want to select the first link in this table, the first one under Application reference. But the data is retrieved from a db so it's difficult to add an attribute to the element, although.... I could probably do something clever with .ejs code. Let me try.

Reply:
you should use test-ids in that case